BDArmory/Radar/RadarUtils.cs

```diff
@@ -209,13 +209,14 @@ private static TargetInfo GetVesselRadarCrossSection(Vessel v, bool force = fals
         /// <summary>
         /// Internal method: get a vessels siganture modifiers (ecm, stealth, ...)
         /// </summary>
-        private static float GetVesselModifiedSignature(Vessel v, TargetInfo ti)
+        public static float GetVesselModifiedSignature(Vessel v, TargetInfo ti)
         {
             ti.radarRCSReducedSignature = ti.radarBaseSignature;
             ti.radarModifiedSignature = ti.radarBaseSignature;
             ti.radarLockbreakFactor = 1;
 
-            //
+            //Wouldn't this make more sense to calculate once when ECM (de)activated, instead of re-calcing every fixedUpdate tick?
```

Comment:
Probably, yeah, unless something in here is changing from frame-to-frame. This is more Josue's area of expertise, so maybe flag it and ask him when he's back?